test/e2e.method.send.js

```diff
@@ -73,7 +73,8 @@ describe('method.send [ @E2E ]', function() {
 
         before(async function(){
             // Ganache runs ws and http over the same port
-            var port = process.env.GANACHE ?  8545 : 8546;
+            // & we use it in headless browser tests
+            var port = ( process.env.GANACHE || global.window ) ?  8545 : 8546;
```

Comment:
same here ``getPort``

Reply:
Added in 11057d00c2fe9bf67ac2625075045a4c77c28800